cmd/incusd/instance_console.go

```diff
@@ -500,6 +521,41 @@ func instanceConsolePost(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(fmt.Errorf("Instance is frozen"))
 	}
 
+	// Find any running 'ConsolShow' operation for the instance.
+	// If the '--force' flag was used, cancel the running operation. Otherwise, notify the user about the operation.
+	for _, op := range operations.Clone() {
+		// Consider only console show operations with Running status.
+		if op.Type() != operationtype.ConsoleShow || op.Project() != projectName || op.Status() != api.Running {
+			continue
+		}
+
+		// Fetch instance name from operation.
+		r := op.Resources()
+		apiUrls := r["instances"]
+		if len(apiUrls) < 1 {
+			return response.SmartError(fmt.Errorf("Operation does not have an instance URL defined"))
+		}
+
+		urlParts := strings.Split(apiUrls[0].URL.Path, "/")
```

Comment:
Would it be better to use [`path.Split`](https://pkg.go.dev/path#Split) here?

Reply:
Fixed.